### PR TITLE
Workflows to calculate an md5sum from a CRAM file

### PR DESCRIPTION
**Pull request by wshands**
CWL and WDL workflows that calculate an md5sum for a CRAM file by converting the CRAM file to a SAM file stripping off the header and passing the file to md5sum.

It also has checker workflows for the WDL and CWL

### CODE REVIEW

File: CRAM-no-header-md5sum/CRAM_md5sum_checker_wrapper.wdl [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import "md5sum/CRAM_md5sum.wdl" as f1
```
Comment (coverbeck) [inline CRAM-no-header-md5sum/CRAM_md5sum_checker_wrapper.wdl]: Does this need to run on FireCloud? If so, don't they only support http(s) imports?

File: CRAM-no-header-md5sum/md5sum/CRAM_tool_samtools.cwl [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    inputBinding:
+      position: 1
+      prefix: -T
+    doc: The genome reference file for the BAM or CRAM file.
+
+  reference_index_file:
+    type: File
+    inputBinding:
+      position: 2
+      prefix: -t
```
Comment (coverbeck) [inline CRAM-no-header-md5sum/md5sum/CRAM_tool_samtools.cwl:48]: Slightly confusing that this prefix and other prefix only differ by case. Could you pick more meaningful values?

Comment (coverbeck) [reply CRAM-no-header-md5sum/md5sum/CRAM_tool_samtools.cwl:48]: Or is this because this is what SAM tools is expecting?

**Review by coverbeck — CHANGES_REQUESTED** [review]
Prefix comment is just a suggestion and may be even be incorrect, so feel free to ignore. It just looks odd to me.

Import statement I think needs to be addressed, unless this is not for FC, in which case, just let me know, and I'll approve.

File: CRAM-no-header-md5sum/CRAM_md5sum_checker_wrapper.wdl (hunk 1 above)
Comment (wshands) [reply CRAM-no-header-md5sum/CRAM_md5sum_checker_wrapper.wdl]: Great thanks for catching this

File: CRAM-no-header-md5sum/md5sum/CRAM_tool_samtools.cwl (hunk 2 above)
Comment (wshands) [reply CRAM-no-header-md5sum/md5sum/CRAM_tool_samtools.cwl:48]: These are samtools parameters http://www.htslib.org/doc/samtools-1.0.html 
-T A FASTA format reference FILE, optionally compressed by bgzip and ideally indexed by samtools faidx. If an index is not present, one will be generated for you.
-t Genome reference index file. If you run: `samtools faidx <ref.fa>', the resulting index file <ref.fa>.fai can be used as this FILE.

**Review by coverbeck — APPROVED** [review]

**Pull request merged**